dartsim/src/JointFeatures.cc

```diff
@@ -89,7 +89,12 @@ void JointFeatures::SetJointAcceleration(
 void JointFeatures::SetJointForce(
     const Identity &_id, const std::size_t _dof, const double _value)
 {
-  this->ReferenceInterface<JointInfo>(_id)->joint->setForce(_dof, _value);
+  auto joint = this->ReferenceInterface<JointInfo>(_id)->joint;
+  if (joint->getActuatorType() != dart::dynamics::Joint::FORCE)
+  {
+    joint->setActuatorType(dart::dynamics::Joint::FORCE);
```

Comment:
@azeey I guess codecov didn't run for #74, but it's now complaining that the patch coverage is too low. This is the only line without coverage. Did you expect this line to be covered by the current tests?

Reply:
Ah, no, I don't think we have a test that uses both SERVO and FORCE actuator types. I can add one to ign-physics1, but I'm not sure I can get to it this week.

Reply:
it's ok; I was just checking